gammapy/estimators/map/ts.py

```diff
@@ -75,6 +76,8 @@ class TSMapEstimator(Estimator, parallel.ParallelMixin):
         Number of sigma for flux error. Default is 1.
     n_sigma_ul : int
         Number of sigma for flux upper limits. Default is 2.
+    n_sigma_sensitivity : int
+        Number of sigma for flux  sensitivity. Default is 5.
```

Comment:
Is it the integral sensitivity above the energy threshold given by the safe_mask or the fit_mask? This energy threshold is a key parameter to give to the user. Is it the case?

Reply:
Argh, damned. I have not checked that... 
In any case, should we implement an utility plot function that give the exact energy threshold? Whatever the origin of the mask, the value should be quoted. If you think that this is necessary, could you make a dedicated feature-request issue?